### PR TITLE
add links and pages schemas, use with list_all_keys

### PR DESCRIPTION
**Pull request by hilary**
https://jira.internal.digitalocean.com/browse/APICLI-403

I validated both endpoints using

`prism proxy DigitalOcean-public.v2.yaml https://api.digitalocean.com/v2 --errors`

then running requests against it with `curl`, e.g., 

`curl -v -H "Authorization: Bearer $DOCTL_TOKEN" http://127.0.0.1:4010/account/keys`
`curl -v -H "Authorization: Bearer $DOCTL_TOKEN" "http://127.0.0.1:4010/account/keys?page=2&per_page=1"`

### CODE REVIEW

File: components/schemas.yml [hunk 1]
```diff
@@ -65,13 +65,15 @@ Account:
...
     - $ref: '#/WrappedSSHKeys'
+    - $ref: 'schemas/pages.yml#/Pagination'
     - $ref: '#/Meta'
   example:
     ssh_keys:
       - id: 512189
         fingerprint: 3b:16:bf:e4:8b:00:8b:b8:59:8c:a9:d3:f0:19:45:fa
         public_key: "ssh-rsa AEXAMPLEaC1yc2EAAAADAQABAAAAQQDDHr/jh2Jy4yALcK4JyWbVkPRaWmhck3IgCoeOO3z1e2dBowLh64QAM+Qb72pxekALga2oi4GvT+TlWNhzPH4V example"
         name: My SSH Public Key
+    links: {}
```
Comment (andrewsomething) [inline components/schemas.yml]: This doesn't seem to override the links examples here. The output of `swagger-cli bundle` still has the examples with `images` in the path. Looks like this works:

```
    - $ref: 'schemas/pages.yml#/Pagination'
      example: {}
```

Comment (hilary) [reply components/schemas.yml]: Nice catch!

**Comment by scotchneat** [comment]
I tested `/account/keys` with an invalid token (intentionally) and received the following error:

```
{
    "detail": "Your request does not fullfil the security requirements and no HTTP unauthorized response was found in the spec, so Prism is generating this error for you.",
    "headers": {
        "WWW-Authenticate": "Bearer"
    },
    "status": 401,
    "title": "Invalid security scheme used",
    "type": "https://stoplight.io/prism/errors#UNAUTHORIZED"
}
```

This might be something we could address in another Jira ticket, but I wanted to note it here while reviewing this PR.

**Comment by hilary** [comment]
@scotchneat yeah, I was planning on doing common responses such as unauthorized in a separate ticket / PR. There's so much in this PR as it is...

Created tickets for 401, 500 and for researching the full set.

File: endpoints/ssh_keys.yml [hunk 2]
```diff
@@ -20,4 +23,42 @@ get:
       content:
         application/json:
           schema:
-            $ref: '../components/schemas.yml#/DecoratedSSHKeys'
\ No newline at end of file
+            $ref: '../components/schemas.yml#/DecoratedSSHKeys'
+          examples:
```
Comment (andrewsomething) [inline endpoints/ssh_keys.yml:27]: Are these examples meant to be here?

Putting the bundled output into SwaggerHub does not show what I'd expect:

image

Comment (hilary) [reply endpoints/ssh_keys.yml:27]: Yes, those examples are meant to be there. You can't put multiple examples on a schema.

I bundled and looked locally. I'll upload to swaggerhub and look.

Comment (hilary) [reply endpoints/ssh_keys.yml:27]: Ah, ok, I wasn't sure how to make overriding work with `allOf`, etc. I was trying it out but it looks like it doesn't resolve correctly. Gonna have to use a different approach for these examples.

Comment (scotchneat) [reply endpoints/ssh_keys.yml:27]: That looks like the bundling didn't properly update the ref to a local component (notice the `../` in the path.

**Review by andrewsomething — APPROVED** [review]
:+1:

**Review by scotchneat — APPROVED** [review]
lgtm

**Pull request merged**